src/Features/JsonPatch.SystemTextJson/src/Converters/OperationConverterOfT.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.JsonPatch.SystemTextJson.Operations;
+
+namespace Microsoft.AspNetCore.JsonPatch.SystemTextJson.Converters;
+
+internal class OperationConverter<T> : JsonConverter<Operation<T>> where T : class
```

Comment:
I think this is currently doing what the default converter in S.T.J. will do. It can be removed if you don't plan on adding special custom serialization/deserialization logic or validation.